readthedocs/api/v3/serializers.py

```diff
@@ -531,7 +544,9 @@ class ProjectSerializerBase(FlexFieldsModelSerializer):
     translation_of = serializers.SerializerMethodField()
     default_branch = serializers.CharField(source='get_default_branch')
     tags = serializers.StringRelatedField(many=True)
-    users = UserSerializer(many=True)
+
+    if not settings.RTD_ALLOW_ORGANIZATIONS:
+        users = UserSerializer(many=True)
```

Comment:
Do we need this `if` here? I think it's unnecessary since we are controlling the fields included in the response from `Meta.fields`.

Reply:
yes, otherwise you get

> The field 'users' was declared on serializer ProjectSerializerBase, but has not been included in the 'fields' option.